proctord/utility/utils.go

```diff
@@ -2,11 +2,22 @@ package utility
 
 const ClientError = "malformed request"
 const ServerError = "Something went wrong"
-const UnauthorizedError = "Unauthorized. Please check the email id and access token"
+
+const UnauthorizedErrorMissingConfig = "EMAIL_ID or ACCESS_TOKEN is not present in proctor config file."
+const UnauthorizedErrorInvalidConfig = "Please check the EMAIL_ID and ACCESS_TOKEN validity in proctor config file."
+const UnauthorizedError = "Please check the EMAIL_ID and ACCESS_TOKEN validity in proctor config file." //to deprecate
```

Comment:
Why is it marked to deprecate? We can replace usage of `UnauthorizedError` with `UnauthorizedErrorHeader` and , `UnauthorizedErrorMissingConfig`, `UnauthorizedErrorInvalidConfig` right?